src/main/java/io/kokuwa/maven/helm/AbstractHelmMojo.java

```diff
@@ -57,39 +57,58 @@ public abstract class AbstractHelmMojo extends AbstractMojo {
 	@Component(role = org.sonatype.plexus.components.sec.dispatcher.SecDispatcher.class, hint = "default")
 	private SecDispatcher securityDispatcher;
 
+	/**
+	 * Controls whether a local binary should be used instead of downloading it.
+	 * If set to `true` path has to be set with property `executableDirectory`.
+	 */
```

Comment:
`javadoc` not supports `markdown` formats

will be appreciated to add `@since <plugin version>` javadoc tag -  when parameter was introduced

Reply:
Please look at https://issues.apache.org/jira/browse/MPLUGIN-420

If you no provide `@since` for parameter - value from Mojo level will be copied to parameter

I hope `m-plugin-p 3.7.0` will be released soon

Reply:
Ok, i'll add it on all parameters